schema/revert/shipping/views.sql

```diff
@@ -1042,6 +1042,7 @@ create or replace view shipping.hcov19_observation_v1 as
         best_available_site as site,
         best_available_site_type as site_type,
 
+        location.hierarchy->'tract' as census_tract,
```

Comment:
did this slip in here accidentally?

Reply:
This is part of the auto-generated revert script where sqitch copies the previous deploy script. I believe this is correct.

Reply:
Oh, right. I almost thought that but for some reason I overlooked that this is a `revert`. Makes sense